File: src/main/kotlin/me/sd_master92/customvoting/database/PlayerData.java
package me.sd_master92.customvoting.database;

import me.sd_master92.customvoting.constants.Voter;

public class PlayerData implements Voter
{
    private final String uuid;
    private final String name;
    private final int votes;
    private final long last;
    private final int queue;

    public PlayerData(String uuid, String name, int votes, long last, int queue)
    {
        this.uuid = uuid;
        this.name = name;
        this.votes = votes;
        this.last = last;
        this.queue = queue;
    }

    public String getName()
    {
        return name;
    }

    public int getQueue()
    {
        return queue;
    }

    public String getUuid()
    {
        return uuid;
    }

    public int getVotes()
    {
        return votes;
    }

    public long getLast()
    {
        return last;
    }
}

File: src/main/kotlin/me/sd_master92/customvoting/database/PlayerData.kt
package me.sd_master92.customvoting.database

import me.sd_master92.customvoting.constants.Voter

class PlayerData(
    override val uuid: String,
    override val name: String,
    override val votes: Int,
    override val last: Long,
    val queue: Int,
    override val period: Int
) : Voter